homeassistant/components/aussie_broadband/sensor.py

```diff
@@ -0,0 +1,140 @@
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, SERVICE_ID
+
+SENSOR_DESCRIPTIONS: tuple[SensorEntityDescription, ...] = (
+    # Internet Services sensors
+    SensorEntityDescription(
+        key="usedMb",
+        name="Data Used",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=DATA_MEGABYTES,
+        icon="mdi:network",
+    ),
+    SensorEntityDescription(
+        key="downloadedMb",
+        name="Downloaded",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=DATA_MEGABYTES,
+        icon="mdi:download-network",
+    ),
+    SensorEntityDescription(
+        key="uploadedMb",
+        name="Uploaded",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=DATA_MEGABYTES,
+        icon="mdi:upload-network",
+    ),
+    # Mobile Phone Services sensors
+    SensorEntityDescription(
+        key="national",
+        name="National Calls",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        icon="mdi:phone",
+    ),
+    SensorEntityDescription(
+        key="mobile",
+        name="Mobile Calls",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        icon="mdi:phone",
+    ),
+    SensorEntityDescription(
+        key="international",
+        name="International Calls",
+        entity_registry_enabled_default=False,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        icon="mdi:phone-plus",
+    ),
+    SensorEntityDescription(
+        key="sms",
+        name="SMS Sent",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        icon="mdi:message-processing",
+    ),
+    SensorEntityDescription(
+        key="internet",
+        name="Data Used",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=DATA_KILOBYTES,
+        icon="mdi:network",
+    ),
+    SensorEntityDescription(
+        key="voicemail",
+        name="Voicemail Calls",
+        entity_registry_enabled_default=False,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        icon="mdi:phone",
+    ),
+    SensorEntityDescription(
+        key="other",
+        name="Other Calls",
+        entity_registry_enabled_default=False,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        icon="mdi:phone",
+    ),
+    # Generic sensors
+    SensorEntityDescription(
+        key="daysTotal",
+        name="Billing Cycle Length",
+        native_unit_of_measurement=TIME_DAYS,
+        icon="mdi:calendar-range",
+    ),
+    SensorEntityDescription(
+        key="daysRemaining",
+        name="Billing Cycle Remaining",
+        native_unit_of_measurement=TIME_DAYS,
+        icon="mdi:calendar-clock",
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+):
```

Comment:
Missing return value typing.
```py
) -> None:
```

Reply:
Someone else beat me to it https://github.com/home-assistant/core/pull/64836